Weather/Views/MapGeolocationView.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  MapGeolocationView.swift
+//  Weather
+//
+//  Created by Alex on 6.09.22.
+//
+
+import SwiftUI
+import MapKit
+
+struct MapGeolocationView: View {
```

Comment:
Why do you need to show map? Can't you just get current coordinates and use them in a request without showing it?